Document reason of ssh key not working (#26)
caprover/caprover-website: docs/deployment-methods.md

## Automatic Deploy using Github, Bitbucket and etc.
This method is perhaps the most convenient one. This method automatically triggers a build with a `captain-definiton` file when you push your repo to a specific branch (like `master` or `staging` or `release` or etc). To setup this, go to your apps settings and enter the repo information:
- repo: This is the main HTTPS address of repo, in case of github, it is in `github.com/someone/something` format. Make sure it does NOT include `https://` prefix and `.git` suffix.
- branch: The branch you want to be tracked, for example `master` or `staging` or `release`...
- github/bitbucket username(email address): This is username that will be used when Captain downloads the repo.
- github/bitbucket password: You can enter any non-empty text, like `123456`, for public projects.

> - Or, instead of username/password, use SSH Key: Be sure that is uses PEM as OPENSSH does not work

Author: ```suggestion
- Or, instead of username/password, use SSH Key: Make sure to use PEM format as other formats may not work
```

## Automatic Deploy using Github, Bitbucket and etc.

> This method is perhaps the most convenient one. This method automatically triggers a build when you push your repo to a specific branch (like `master` or `staging` or `release` or etc). To setup this, go to your apps settings and enter the repo information:

Author: ```suggestion
This method is perhaps the most convenient one. This method automatically triggers a build with a `captain-definiton` file when you push your repo to a specific branch (like `master` or `staging` or `release` or etc). To setup this, go to your apps settings and enter the repo information:
```

- Project > Settings > Webhooks > Add Webhook > Title: Captain Server, URL: Captain Webhook from your apps page.

#### GitLab and Others

> Webhooks can be added in a similar fashion. As long as the webhook fires a POST request, CapRover is able to pick it up and starts a build from the latest commit on the specified branch.

Author: Dont know why it highlights this. Have nothing changed here

Reviewer: It happened for me as well - presumably when you edit through GitHub directly, it changes the EOF format
